### ISSUE TITLE
Clockwork install doesn't set symlink up properly (windows/msys and mac)

### ISSUE THREAD

**Issue by wilsonk** [tauraamui/lilly#267]
I just updated everything here and things were smooth on my Arch Linux clone, but the clockwork symlinking didn't work on windows/msys and mac.

I ended up just symlinking from ~/.local/share/clockwork/build/clockwork to a link in my path and it works fine...but I just thought I would mention it.

Seems like the clockwork/scripts/symlink.vsh file isn't working correctly on these two platforms?

**Comment by wilsonk** [comment]
Ah, I see it is an issue upstream, as clockwork doesn't seem to be tested or made specifically for mac/msys. I guess we should fix and make a PR for the project?

**Comment by tauraamui** [comment]
Yeah that would be a good idea. I happened to encounter the clockwork project right around when I decided we should either:
a): write a custom VSH script file to be used for all of our "actions/tasks/commands/recipes"
b): create a custom tool which is written in V that behaves like just/make

The design of clockwork and how it works is good, my goal for migrating to it was to hopefully provide an easier experience for developers with the V compiler but nothing else. Kind of how I found `just` very useful and easy to use and install for my Go project workflows (since it's implemented in Go so its just a `go install` command away.

The downside of using such a project however is it is brand new, I was able to request the author implement a bunch of things early on which I didn't have the time to do, for example being able to define task categories and adjust the verbosity of the list output via the config file.
